docs/Preparing_your_workstation.adoc

```diff
@@ -115,8 +121,8 @@ brew install awscli
 mkdir ~/.aws
 cat << EOF >>  ~/.aws/credentials
 [default]
-aws_access_key_id = AKIAJAAYOURACCESSKEY
-aws_secret_access_key = rT54UYOURSECRETACCESSKEY
+aws_access_key_id = YOUR_AWS_ID
```

Comment:
Nit: I'd rather these said something like `changeme`. ID & Password are not really accurate here.

Reply:
done